src/Query/Aggregation/Limit.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query\Aggregation;
+
+use TBolier\RethinkQL\Message\MessageInterface;
+use TBolier\RethinkQL\Query\AbstractQuery;
+use TBolier\RethinkQL\Query\QueryInterface;
+use TBolier\RethinkQL\RethinkInterface;
+use TBolier\RethinkQL\Types\Term\TermType;
+
+class Limit extends AbstractQuery
```

Comment:
Should this inherit from `AbstractAggregation`? Because you can do:

`filter()->orderBy()->limit()` and `filter()->limit()->orderBy()`